test/run

```diff
@@ -192,7 +201,14 @@ check_result $?
 for app in ${WEB_APPS[@]}; do
   prepare ${app}
   run_s2i_build ${app}
-  check_result $?
+  RESULT=$?
+  if [[ "$app" == "pipenv-and-micropipenv-fails-test-app" ]]; then
```

Comment:
I wonder, how about somehow adding the expected result to the test name instead (eg `pipenv-and-micropipenv-should-fail-app`)? Would make it easier for future expected failure test cases to be added.

Reply:
Good idea, I'm gonna implement it.